### PR TITLE
{perf,toolchain}[gompi/2021.07,system/system] gompi v2021.07, OSU-Micro-Benchmarks v5.7.1

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

Candidate for `gompi/2021b`

* [x]  #13668

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
jsfl01.jusuf - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/14b62f802548363ef6e3501836eea904 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/5cc137f3fe30368916d7c282c71612e2 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
jwvis00.juwels - Linux centos linux 8.4.2105, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/07c84c3172914ebc8151283e88a4f571 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/e24c8dc123b790719305ee7e9a14ed63 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0305u15a.bear.cluster - Linux RHEL 8.4, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/9139303b9eab2d91d52c9b24c2ada745 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u12a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/4c671c485ec1597fa92043c6fb9093dd for a full test report.

File: easybuild/easyconfigs/g/gompi/gompi-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'Toolchain'
+
+name = 'gompi'
+version = '2021b'
```
Comment (boegel) [inline easybuild/easyconfigs/g/gompi/gompi-2021b.eb:4]: Should we name this `2021.07` until we're confident with the version choices?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/gompi/gompi-2021b.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/g/gompi/gompi-2021b.eb]: Yes. Done in d6f90fc and 3485534

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3585.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a6fc5ccb9d8766ddb81d33eab10972a1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (2 easyconfigs in total)
node3111.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/52de76ad68e74967cdb1c672eb2e64d8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/7ba3ea7bf4f2dde0bbddec7b7df60cad for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/f72c3f7a7f608d638751814a80f32160 for a full test report.

**Pull request merged**